core/domain/story_domain.py

```diff
@@ -297,16 +301,19 @@ def validate(self):
 class StoryContents(object):
     """Domain object representing the story_contents dict."""
 
-    def __init__(self, story_nodes, starting_node_id):
+    def __init__(self, story_nodes, starting_node_id, next_node_id):
         """Constructs a StoryContents domain object.
 
         Args:
             story_nodes: list(StoryNode). The list of story nodes that are part
                 of this story.
             starting_node_id: str. The id of the starting node of the story.
+            next_node_id: int. The index for the next node to be added to the
+                story.
         """
         self.starting_node_id = starting_node_id
```

Comment:
I thought this was becoming the initial_node_id

Reply:
Done.